src/sql/workbench/contrib/query/browser/gridPanel.ts

```diff
@@ -726,7 +726,7 @@ export abstract class GridTableBase<T> extends Disposable implements IView {
 						graphFileType: result.queryExecutionPlanFileExtension
 					};
 
-					const executionPlanInput = this._register(this.instantiationService.createInstance(ExecutionPlanInput, undefined, executionPlanGraphInfo));
+					const executionPlanInput = this.instantiationService.createInstance(ExecutionPlanInput, undefined, executionPlanGraphInfo);
```

Comment:
do we still need to dispose the input somehow?  or it get's cleaned up by the editorService?

Reply:
The inputs are disposed whenever the editor tab associated with it is closed. So, no cleanups are required on this end of the code.